framework/src/modules/dpos_v2/commands/unlock.ts

```diff
@@ -39,6 +39,9 @@ export class UnlockCommand extends BaseCommand {
 
 	public addDependencies(args: UnlockCommandDependencies) {
 		this._tokenAPI = args.tokenAPI;
+	}
+
+	public init(args: { tokenIDDPoS: TokenIDDPoS }) {
```

Comment:
here also tokenID is not available in the dependencies

Reply:
This is added in https://github.com/LiskHQ/lisk-sdk/pull/6979

Reply:
👍  ill merge #6979 first